code/modules/reagents/chemistry/holder.dm

```diff
@@ -555,7 +799,23 @@
 	if(!D)
 		WARNING("[my_atom] attempted to add a reagent called '[reagent]' which doesn't exist. ([usr])")
 		return FALSE
-	
+
+	if (D.id == "water") //Do like an otter, add acid to water.
```

Comment:
Won't that cause a huge overhead checking every instance of water's pH to see if it's below 2?

Reply:
To clarify, if I made it a reaction, it's reagents would just be water, since when you add water to anything at all and it's acidic, it explodes (in real life), hence the "do like ad otter, add acid to water" verse that chemists say. I've had this happen once irl where I added water to clean a beaker that had reported 1M HCl in it, as I got a ton of burning, caustic gas in my face, as the acid was actually 16M HCl. Lets just say the sink's pipe were incredibly clean after that day.

Reply:
Is there a way I can hijack the add_reagent proc for specifically water?

Reply:
I don't think trying to use the reaction proc to check is a good idea either, because then I can't check to see if you've added water to acid, and not acid to water. It specially has to erupt only when water is added to an acidic solution.

Reply:
If someone can think of a better way to check if the pH of the beaker is less than 2 when adding water to a holder, please say so! I'll try to think it over over the next few days.